docs/contributing/docs-templates.md

```diff
@@ -21,7 +22,19 @@ Here are templates (models) to follow when contributing to Gatsby docs to ensure
 
 Reference guide articles cover discrete topics as documentation with links to other resources. A reference guide explains a Gatsby concept or technique without the step-by-step context provided by a tutorial or recipe.
 
-Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios.
+Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios. Reference material would list text headings for each section of a guide, rather than "steps 1-5" like a tutorial.
```

Comment:
```suggestion
Reference guide sections provide canonical information on how and why to build things with Gatsby for a variety of scenarios. Reference material should list text headings for each section of a guide, rather than "steps 1-5" like a tutorial.
```